OlinkAnalyze/R/olink_normalization_utils.R

```diff
@@ -1046,7 +1052,11 @@ olink_norm_input_cross_product <- function(lst_df,
           ),
         by = stats::setNames(object = "OlinkID_E3072", nm = l_3k_oid_rename),
         relationship = "many-to-one"
```

Comment:
Makes sense why this is kept as many-to-one. Only 1 row for each HT correlation assay (filtered to one block per correlation assay). Looks good.